internal/workload/podman/podman.go

```diff
@@ -282,14 +283,13 @@ func (p *podman) UpdateSecret(name, data string) error {
 	return p.CreateSecret(name, data)
 }
 
-func (p *podman) GenerateSystemdService(podName string, monitoringInterval uint) (service.Service, error) {
-	useName := true
-	report, err := generate.Systemd(p.podmanConnection, podName, &generate.SystemdOptions{UseName: &useName, RestartSec: &monitoringInterval})
+func (p *podman) GenerateSystemdService(podId string, monitoringInterval uint) (service.Service, error) {
+	report, err := generate.Systemd(p.podmanConnection, podId, &generate.SystemdOptions{RestartSec: &monitoringInterval})
 	if err != nil {
 		return nil, err
 	}
 
-	svc, err := service.NewSystemd(podName, "pod-", report.Units)
+	svc, err := service.NewSystemd(podId, "pod-", report.Units)
```

Comment:
From what I understand, `podId` will become part of the service name (`pod-<podId>.service`), which will become pretty lengthy; won't that be a problem for systemd? I don't thing it would matter a lot for debuggability.

Reply:
I've tested and it looks like:
```bash
systemctl status pod-ee7b06ed00c24e66528121a2cd7c1d721e7897a99ad25145d63c4cc529427e0b.service
```
so yes it's lenghty, but works.